packages/js/components/src/experimental-tree-control/typings/global.d.ts

```diff
@@ -0,0 +1,9 @@
+import * as components from '@wordpress/components';
+
+declare module '@wordpress/components' {
+	declare namespace CheckboxControl {
+		interface Props {
+			indeterminate?: boolean;
```

Comment:
Because for [@wordpress/components@19.8.5](https://github.com/WordPress/gutenberg/blob/%40wordpress/components%4019.8.5/packages/components/src/checkbox-control/index.js) there is not a TS definition for the prop `indeterminate`. Also there is not such definition even in any version of `@types/wordpress__components`